x-pack/plugins/task_manager/server/mocks.ts

```diff
@@ -38,7 +40,29 @@ const createStartMock = () => {
   return mock;
 };
 
+const createTaskMock = (overrides: Partial<ConcreteTaskInstance> = {}): ConcreteTaskInstance => {
```

Comment:
New re-usable mock function to mock a task document. This code was repeated in a few unit test files (not anymore!).